<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Vaibhav-HP" failures="3" tests="6" name="Test" time="74.232" errors="0" timestamp="2024-09-20T14:55:47 IST">
  <testcase classname="testCases.TC_LoginDDT" name="verify_loginDDT" time="11.05">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC_LoginDDT.verify_loginDDT(TC_LoginDDT.java:67)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC_LoginDDT" name="verify_loginDDT" time="11.755">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC_LoginDDT.verify_loginDDT(TC_LoginDDT.java:67)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC_LoginDDT" name="verify_loginDDT" time="11.879"/>
  <testcase classname="testCases.TC_LoginDDT" name="verify_loginDDT" time="11.113"/>
  <testcase classname="testCases.TC_LoginDDT" name="verify_loginDDT" time="11.897">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC_LoginDDT.verify_loginDDT(TC_LoginDDT.java:67)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC_LoginDDT" name="verify_loginDDT" time="11.833"/>
</testsuite> <!-- Test -->
